website/components/PropTable.tsx

```diff
@@ -0,0 +1,203 @@
+import React from 'react';
+import { css } from 'emotion';
+import { Table, TableHeader, Row, Cell } from '@leafygreen-ui/table';
+import { Subtitle, InlineCode } from '@leafygreen-ui/typography/dist';
+import PropDefinition from 'components/PropDefinition';
+import TypographyPropTable from 'components/TypographyPropTable';
+
+const subtitleBottomMargin = css`
+  margin-bottom: 24px;
+`;
+
+const tableBottomMargin = css`
+  margin-bottom: 56px;
+`;
+
+const tableHeaderNames: Array<'prop' | 'type' | 'description' | 'default'> = [
+  'prop',
+  'type',
+  'description',
+  'default',
+];
+
+/**
+ * Syntactic units in unist syntax trees are called nodes.
+ * Interface derived from Unified types
+ * See here: https://github.com/syntax-tree/unist#:~:text=Syntactic%20units%20in%20unist%20syntax,and%20implement%20the%20Node%20interface.
```

Comment:
link not working for me perhaps this instead?
```suggestion
 * See here: https://github.com/syntax-tree/unist#nodes
```

Reply:
Do we need to define all of this ourselves? Looks like there's types published here
https://www.npmjs.com/package/@types/unist

Reply:
Yep, you're right